compiler/rustc_interface/src/tests.rs

```diff
@@ -614,6 +614,7 @@ fn test_codegen_options_tracking_hash() {
     tracked!(control_flow_guard, CFGuard::Checks);
     tracked!(debug_assertions, Some(true));
     tracked!(debuginfo, DebugInfo::Limited);
+    tracked!(dwarf_version, Some(5));
```

Comment:
I don't think this and the other DWARF changes were intended to be included?

Reply:
This is a false positive because I previously included upstream commits by accident. Rebasing on the current master branch fixed this.